src/dfastbe/bank_erosion.py

```diff
@@ -374,8 +376,8 @@ def _prepare_initial_conditions(self, config_file, bank_km_mid, zfw_ini) -> Tupl
                 "Erosion", "BankType", bank_km_mid, default=0, ext=".btp"
             )
```

Comment:
can you move the `taucls`, and the `taucls_str` values to the `ErosionInputs` class and make them optional value for the parameters.